core/src/main/java/org/elasticsearch/action/admin/cluster/node/info/PluginInfo.java

```diff
@@ -178,6 +177,10 @@ public int hashCode() {
         return name.hashCode();
     }
 
+    public int compare(PluginInfo other) {
+        return this.name.compareTo(other.name);
+    }
```

Comment:
For the record, I was a bit confused at that method, which I first read as if it was Comparable.compareTo while it's not